Lab2.ipynb

```diff
@@ -0,0 +1,879 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pois5 = poisson.pmf(x, X)\n",
+    "for i in range(4):\n",
+    "    pois5 = signal.convolve(pois1, pois5)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x5 = np.linspace(poisson.ppf(0.01, X),\n",
+    "              poisson.ppf(0.99, X), 61)\n",
+    "ax.plot(x5, pois5, 'b-', ms=8, label='poisson pmf')\n",
+    "plt.yscale('log')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 56,
+   "id": "c0525536-6071-4f53-9288-ac1b4262e64d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# From the log plot, we can see that although the \n",
+    "# log plot of the 5-day sum of the poisson is almost\n",
+    "# a upside-down parabola, it leans toward one side clearly.\n",
+    "# Hence, the current distribution is still a poisson."
+   ]
```

Comment:
I can clearly see the Poisson distribution pattern from the graph you created! However, can you also explain the reasoning in a conceptual/mathematical view?